inc/classes/subscriber/Optimization/class-lazyload-subscriber.php

```diff
@@ -452,6 +452,14 @@ private function can_lazyload_images() {
 	 * @return boolean
 	 */
 	private function can_lazyload_iframes() {
+		if ( ! $this->options->get( 'lazyload_youtube' ) ) {
+			return false;
+		}
```

Comment:
This is a sub-option of the `lazyload_iframes`, so it should be checked separately only when needed